docs/source/quick_start.rst

```diff
@@ -0,0 +1,193 @@
+
+
+OpenMP
+~~~~~~
+::
+
+  export INSTALL_DIR = ${MY_INSTALL_DIR}
+
+  cmake \
+   -DKokkos_ARCH_NATIVE=ON \
+   -DKokkos_ENABLE_OPENMP=ON \
+   -DKokkos_ENABLE_SERIAL=ON \
+   -DCMAKE_CXX_STANDARD=17 \
+   -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+   -DCMAKE_BUILD_TYPE=Release ..
+
+
+CUDA
+~~~~
+
+::
+
+  export INSTALL_DIR = ${MY_INSTALL_DIR}
+
+  cmake \
+    -DKokkos_ARCH_NATIVE=ON \
+    -DKokkos_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE=OFF \
+    -DKokkos_ENABLE_CUDA=ON \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+.. note::
+
+  VEGA90A is for AMD MI200
+  AMD_GFX942 is for AMD MI300
+
+HIP
+~~~
+::
+
+  cmake \
+    -DCRAYPE_LINK_TYPE=dynamic \
+    -DKokkos_ENABLE_TESTS=ON \
+    -DKokkos_ENABLE_HIP=ON \
+    -DKokkos_ARCH_AMD_GFX942=OFF \
+    -DKokkos_ARCH_VEGA90A=ON \
+    -DKokkos_ENABLE_SERIAL=ON \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DCMAKE_CXX_COMPILER=$(which hipcc) \
+    -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+    -DCMAKE_BUILD_TYPE=Release ..
+
+----------------------------------------------
+Basic Configuration Examples for Spack Builds
+----------------------------------------------
+
+*  A number of Kokkos variants can be built / installed via Spack.  You will need to select a variant for the desired backend, and appropriate GPU microarchitecture.   
+*  To explore the range of variants for a package, ``spack info kokkos``, ``spack info trilinos``, etc.
+* `Basic Spack Installation <https://spack.readthedocs.io/en/latest/getting_started.html>`_
+
+
+
+.. note::
+
+  Before installing, you can ``spack spec``  variants to verify the build type.
+
+Serial
+~~~~~~~
+
+::
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +serial cxxstd=20
+
+OpenMP
+~~~~~~
+
+::
+
+  spack spec kokkos@4.2 %gcc@10.3.0 +openmp cxxstd=20
+
+
+CUDA
+~~~~
+
+:: 
+  
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +cuda cuda_arch=70 cxxstd=20 +cuda_relocatable_device_code
+
+
+HIP
+~~~
+
+::
+
+  spack spec / install kokkos@4.2 %gcc@12.2.0 +rocm amdgpu_target=gfx90a cxxstd=20
+
+
+--------------------
+Additional Resources
+--------------------
+
+* `CMake Keywords <https://kokkos.org/kokkos-core-wiki/keywords.html>`_
+* `Building Kokkos <https://kokkos.org/kokkos-core-wiki/building.html>`_
+* `Spack Kokkos Build <https://kokkos.org/kokkos-core-wiki/building.html#spack>`_
+
+
+---
+FAQ
+---
+
+* How to Join Kokkos Slack?
+
+You can find the Slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We automatically whitelist emails from most organizations, but if your email address is not whitelisted, you can contact the Kokkos maintainers (their emails are in the LICENSE file).
+
+
+* How to build for more recent C++??
+
+When configuring Kokkos with CMake (see configruation, add the flag ``-DCMAKE_CXX_STANDARD=20`` (or ``23``). Ensure that the flag is also set for any downstream applications.
+
```

Comment:
Why are we repeating the content of the FAQ?
Did you just copy the file and forgot to remove?

Reply:
I removed it.  A little redundancy is not bad, but in subsequent edits, I agree with you -- it just clutters.